### PR TITLE
Added Description for Localization in installer

### PR DESCRIPTION
**Pull request by SinghRajenM**
Added description for "Localization" on component selection page in installer, because - 
1. As of now there is no description for localization (It show empty if move over happen direct exactly on Localization)
2. It shows wrong description - Either it shows description from plugins or themes (in both ways -> 1. using mouse 2. using keyword)

### CODE REVIEW

File: PowerEditor/installer/nppSetup.nsi [hunk 1]
```diff
@@ -144,9 +144,10 @@ ${MementoSectionDone}
   !insertmacro MUI_FUNCTION_DESCRIPTION_BEGIN
     !insertmacro MUI_DESCRIPTION_TEXT ${explorerContextMenu} 'Explorer context menu entry for Notepad++ : Open whatever you want in Notepad++ from Windows Explorer.'
     !insertmacro MUI_DESCRIPTION_TEXT ${autoCompletionComponent} 'Install the API files you need for the auto-completion feature (Ctrl+Space).'
-				!insertmacro MUI_DESCRIPTION_TEXT ${Plugins} 'You may need those plugins to extend the capacity of Notepad++.'
+    !insertmacro MUI_DESCRIPTION_TEXT ${Plugins} 'You may need those plugins to extend the capacity of Notepad++.'
```
Comment (SinghRajenM) [inline PowerEditor/installer/nppSetup.nsi]: Also, should I change "capacity" to "capabilities"?

File: PowerEditor/installer/nppSetup.nsi [hunk 2]
```diff
@@ -144,9 +144,10 @@ ${MementoSectionDone}
...
     !insertmacro MUI_DESCRIPTION_TEXT ${autoCompletionComponent} 'Install the API files you need for the auto-completion feature (Ctrl+Space).'
-				!insertmacro MUI_DESCRIPTION_TEXT ${Plugins} 'You may need those plugins to extend the capacity of Notepad++.'
+    !insertmacro MUI_DESCRIPTION_TEXT ${Plugins} 'You may need those plugins to extend the capacity of Notepad++.'
+    !insertmacro MUI_DESCRIPTION_TEXT ${localization} 'To use Notepad++ in your favorite language(s), install all/selected langauage(s).'
     !insertmacro MUI_DESCRIPTION_TEXT ${Themes} 'The eye-candy to change visual effects. Use Theme selector to switch among them.'
-				!insertmacro MUI_DESCRIPTION_TEXT ${AutoUpdater} 'Keep your Notepad++ update: Check this option to install an update module which searches Notepad++ update on Internet and install it for you.'
+    !insertmacro MUI_DESCRIPTION_TEXT ${AutoUpdater} 'Keep your Notepad++ update: Check this option to install an update module which searches Notepad++ update on Internet and install it for you.'
```
Comment (SinghRajenM) [inline PowerEditor/installer/nppSetup.nsi]: Should we consider changing 
"Keep your Notepad++ update" as "Keep your Notepad++ up-to-date" or "Keep your Notepad++ updated"?

**Comment by SinghRajenM** [comment]
Screenshot
installer

File: PowerEditor/installer/nppSetup.nsi (hunk 2 above)
Comment (dail8859) [inline PowerEditor/installer/nppSetup.nsi]: Either "up-to-date" or "updated" would be fine, and "your" seems unneeded.

There are several other minor updates that should be made:

"Keep Notepad++ updated: Check this option to install an update module which searches *for* Notepad++ update*s* on *the* Internet and install*s* it for you."

Personally I find this sentence overly verbose. I prefer something like:

"Keep Notepad++ updated: Automatically download and install the latest updates."

File: PowerEditor/installer/nppSetup.nsi [hunk 3]
```diff
@@ -144,10 +144,10 @@ ${MementoSectionDone}
   !insertmacro MUI_FUNCTION_DESCRIPTION_BEGIN
     !insertmacro MUI_DESCRIPTION_TEXT ${explorerContextMenu} 'Explorer context menu entry for Notepad++ : Open whatever you want in Notepad++ from Windows Explorer.'
     !insertmacro MUI_DESCRIPTION_TEXT ${autoCompletionComponent} 'Install the API files you need for the auto-completion feature (Ctrl+Space).'
-    !insertmacro MUI_DESCRIPTION_TEXT ${Plugins} 'You may need those plugins to extend the capacity of Notepad++.'
-    !insertmacro MUI_DESCRIPTION_TEXT ${localization} 'To use Notepad++ in your favorite language(s), install all/selected langauage(s).'
+    !insertmacro MUI_DESCRIPTION_TEXT ${Plugins} 'You may need these plugins to extend the capabilities of Notepad++.'
+    !insertmacro MUI_DESCRIPTION_TEXT ${localization} 'To use Notepad++ in your favorite language(s), install all/desired langauage(s).'
```
Comment (MAPJe71) [inline PowerEditor/installer/nppSetup.nsi]: `language(s)`

Comment (SinghRajenM) [inline PowerEditor/installer/nppSetup.nsi]: Corrected!

**Pull request closed without merging**